src/control/server/README.md

```diff
@@ -0,0 +1,351 @@
+
+Log file and level mask for both data (`daos_io_server`) and control (`daos_server`) planes can be set in the server config file.
+TODO: examples for global (control) and per-server (data) log parameters in config including DAOS specific syntax (D_LOG_MASK=ERR,mgmt=DEBUG).
+
+### Selecting and formatting NVMe/block device storage through the config file
+
+Parameters prefixed with `nvme_` in the per-server section of the config file determine how NVMe storage will be assigned for use by DAOS on the storage node.
+TODO: examples for NVMe and AIO (emulation) classes including config file syntax and results/logging of formatting operations including generated nvme.conf file and SPDK device format through language bindings.
+
+### Selecting and formatting SCM/pmem storage through the config file
+
+Parameters prefixed with `scm_` in the per-server section of the config file determine how SCM storage will be assigned for use by DAOS on the storage node.
+TODO: examples for both DCPM and RAM (emulation) SCM classes including config file syntax and results/logging of formatting operations.
+
+## Subcommands
+
+`daos_server` supports various subcommands (see `daos_server --help` for available subcommands) which will perform stand-alone tasks as opposed to launching as a daemon (default operation if launched without subcommand).
+
+### prep-nvme
+
+This subcommand requires elevated permissions and needs to be run with root permissions (sudo).
+
+NVMe access through SPDK as an unprivileged user can be enabled by first running `sudo daos_server prep-nvme -p 4096 -u bob`. This will perform the required setup in order for `daos_server` to be run by user "bob" who will own the hugepage mountpoint directory and vfio groups as needed in SPDK operations. If the `target-user` is unspecified (`-u` short option), the target user will be the issuer of the sudo command (or root if not using sudo). The specification of `hugepages` (`-p` short option) defines the number of huge pages to allocate for use by SPDK.
+
+The configuration commands that require elevated permissions are in `src/control/mgmt/init/setup_spdk.sh` (script is installed as `install/share/setup_spdk.sh`).
+
+The sudoers file can be accessed with command `visudo` and permissions can be granted to a user to execute a specific command pattern (requires prior knowledge of `daos_server` binary location):
+```
+linuxuser ALL=/home/linuxuser/projects/daos_m/install/bin/daos_server prep-nvme*
+```
+
+See `daos_server prep-nvme --help` for usage.
+
+### show-storage
+
+List NVMe SSDs and SCM modules locally attached to the host.
+
+See `daos_server show-storage --help` for usage.
+
+## Shell Usage
+
+In order to run the shell to perform administrative tasks, build and run the `daos_server` as per the [quickstart guide](https://github.com/daos-stack/daos/blob/master/doc/quickstart.md).
+
+`daos_server` is to be run as root in order to perform administrative tasks, to be run through `orterun` as root:
```

Comment:
redundancy - "to be run as root" and "to be run through orterun as root" in the same sentence